lib/commands/bootstrap.js

```diff
@@ -15,8 +15,11 @@ module.exports = {
 	 * @param {Object} data
 	 * @param {Object} data.options Additional options provided by the user.
 	 * @param {String} data.options.recursive Whether to install dependencies of packages.
-	 * @param {String} data.options.repositoryResolver Module which will resolve repositories for packages.
 	 * @param {String} data.name Name of current package that will be parsed.
+	 * @param {Object|null} data.repository
```

Comment:
Please extract this to some typedef. Repeating documentation, as well as the code, is bad.

You can put that typedef in `parserepositoryurl.js`.